src/App.js

```diff
@@ -21,13 +57,29 @@ function App() {
       [prefersDarkMode],
   );
 
+  function addStock(title) {
+    //add stock from search bar
+    //FIXME: should provide necessary information of stock here to pop up on card
+    let data = {title: title, id: Math.random()}
```

Comment:
Using a random float as the ID isn't currently causing issues since we just use it to check for equality, but it might be more robust to store a random unique integer (or a hash of the title for extra fanciness).